gui/src/system/SystemPages.tsx

```diff
@@ -332,9 +336,17 @@ function deleteConfigEntry(entry: Mapping) {
   console.log(editableEntries());
 }
 
+
+function clear_configuration_error() {
+  setCurrentConfigurationErrorCode('');
```

Comment:
Is there a reason here why this should be an empty string instead of a value like `null`?